src/app/providers.tsx

```diff
@@ -0,0 +1,10 @@
+'use client';
+import { ThemeProvider } from 'next-themes';
```

Comment:
Is this working for you?  
I am getting a _Cannot find module 'next-themes' or its corresponding type declarations._. 🤔

Reply:
Below is a screenshot of the compile-time error and the console output when I attempt to run the app locally.

![image](https://github.com/map-of-pi/map-of-pi-frontend-react/assets/72060658/d63ec6f7-33af-4921-b748-8f611bb161b7)